auditlog/models.py

```diff
@@ -1,18 +1,22 @@
 import ast
 import json
+from typing import Any, Dict, List
 
 from dateutil import parser
 from dateutil.tz import gettz
 from django.conf import settings
 from django.contrib.contenttypes.fields import GenericRelation
 from django.contrib.contenttypes.models import ContentType
+from django.core import serializers
 from django.core.exceptions import FieldDoesNotExist
 from django.db import DEFAULT_DB_ALIAS, models
 from django.db.models import Q, QuerySet
 from django.utils import formats, timezone
 from django.utils.encoding import smart_str
 from django.utils.translation import gettext_lazy as _
 
+from .diff import mask_str
```

Comment:
Please use absolute imports, relative imports are not used in this repository.

Reply:
👍  done.